packages/clerk-auth/src/index.ts

```diff
@@ -1,19 +1,22 @@
 import { createClerkClient } from '@clerk/backend'
 import type { ClerkClient, ClerkOptions } from '@clerk/backend'
+import type { SignedInAuthObject, SignedOutAuthObject } from '@clerk/backend/internal'
+import type { PendingSessionOptions } from '@clerk/types'
 import type { Context, MiddlewareHandler } from 'hono'
 import { env } from 'hono/adapter'
 
-type ClerkAuth = ReturnType<Awaited<ReturnType<ClerkClient['authenticateRequest']>>['toAuth']>
+type GetAuthOptions = PendingSessionOptions
 
 declare module 'hono' {
   interface ContextVariableMap {
     clerk: ClerkClient
-    clerkAuth: ClerkAuth
+    clerkAuth: (options?: GetAuthOptions) => SignedInAuthObject | SignedOutAuthObject | null
```

Comment:
This was originally `AuthObject` type but we updated the `AuthObject` type to include machine auth types. This change here is for backwards compatibility.

More info: https://github.com/clerk/javascript/pull/5689